install

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eo pipefail;
+[[ $DOKKU_TRACE ]] && set -x
+
+
+echo "%dokku ALL=(ALL) NOPASSWD: /bin/cp, /usr/sbin/service cron restart" > /etc/sudoers.d/dokku-supply-config
+chmod 0440 "/etc/sudoers.d/dokku-supply-config"
```

Comment:
@josegonzalez I've had a look at the nginx and postgres plugin and it seems like this is what's needed. However, when I try to install the plugin the `dokku-supply-config` is not created. I'm not sure exactly what I'm missing here.